pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[build-system]
+requires = ["setuptools>=61.2", "setuptools_scm[toml]>=3.4.3"]
+build-backend = "setuptools.build_meta"
```

Comment:
I don't think we need the features of `setuptools_scm`, and I'd generally like us to switch to `hatchling` as we did in python-tuf and in-toto (https://github.com/theupdateframework/python-tuf/pull/1896, https://github.com/in-toto/in-toto/pull/490).

Reply:
Oh right, I forgot that we don't have the version in `__init__.py`. I'd rather add it there and not use git to infer it. But I'm fine with doing that in a follow up PR.